internal/openmetrics-exporter/pods_space_collector.go

```diff
@@ -129,6 +144,12 @@ func NewPodsSpaceCollector(fa *client.FAClient) *PodsSpaceCollector {
 			[]string{"name", "space"},
 			prometheus.Labels{},
 		),
+		MediatorDesc: prometheus.NewDesc(
+			"purefa_pod_mediator_status",
+			"FlashArray pod space in bytes",
```

Comment:
This doesn't seem like the correct description

Reply:
changed with the proper description